### PR TITLE
WIP: add Tailwind CSS 2.1.0

### PR DESCRIPTION
**Pull request by godbout**

### CODE REVIEW

**Comment by godbout** [comment]
ah. was expecting the ci again to see if i didn't screw up anything. oops.

File: docsets/Tailwind_CSS/docset.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   ],
   "specific_versions": [
     {
-      "version": "2.0.x/14_2020-11-22",
+      "version": "2.1.x/15_2021-04-07",
       "archive": "versions/2.0.x/tailwindcss.tgz"
```
Comment (Kapeli) [inline docsets/Tailwind_CSS/docset.json:17]: You need to change the path to `versions/2.1.x/tailwindcss.tgz` I think.

**Comment by Kapeli** [comment]
Validation issues:
```
File not found at "tailwindcss.com/https://github.com/tailwindlabs/tailwindcss-forms" for entry with row id "178" of "tailwindcss.docset"
File not found at "tailwindcss.com/https://github.com/tailwindlabs/tailwindcss-aspect-ratio" for entry with row id "179" of "tailwindcss.docset"
File not found at "tailwindcss.com/https://github.com/tailwindlabs/tailwindcss-line-clamp" for entry with row id "180" of "tailwindcss.docset"
```

Please note that you can validate locally too. Run this command with the working dir set to the base of the repo:
```
wget http://kapeli.com/feeds/zzz/docsetcontrib.tgz && tar -xzf docsetcontrib.tgz && ./docsetcontrib --verify
```

**Comment by godbout** [comment]
i just launched the tweets. damn. 

thanks.

File: docsets/Tailwind_CSS/docset.json (hunk 1 above)
Comment (godbout) [reply docsets/Tailwind_CSS/docset.json:17]: wth. and i'm not even drunk.

**Comment by godbout** [comment]
`Missing docset.json in "Alfred_4"`

seems i've screwed up another one of my repo in the past. excellent.

**Comment by godbout** [comment]
hmm ok so there's four links that point to GitHub repos and `wget` rewrote only three of them. nice.

**Comment by godbout** [comment]
> ```
> wget http://kapeli.com/feeds/zzz/docsetcontrib.tgz && tar -xzf docsetcontrib.tgz && ./docsetcontrib --verify
> ```

that command is cool. i think i'll come back here often.

**Comment by Kapeli** [comment]
Ready to merge? It passes verification now.

**Comment by godbout** [comment]
> Ready to merge? It passes verification now.

well, yes. i've removed the culprits from the Guide entries, compiled the whole thing, and ran the magic command. all seemed fine. i would say yes, merge, but also be aware that right after you merge (it may be in a day) i will probably find an issue about ten seconds after. but i would still say yes, please merge. thank you!

**Comment by Kapeli** [comment]
Great. Thank you! 👍

**Pull request merged**